### ISSUE TITLE
Application does not support private X.509 PKI

### ISSUE THREAD

**Issue by astravelata** [phpbg/easysync#15]
**Describe the bug**
EasySync does not appear to read user-defined certificate authorities from the Android system. Trying to connect to a server that uses a private CA results in:

```
Error: java.security.cert.CertPathValidatorException: Trust anchor for certification path not found.
```

This certificate works in Chrome and other applications that read the CA, so appears to be limited to EasySync. EasySync should preferably read from the user's trusted CAs, but could also allow for an "insecure" option in which the user disables TLS validation (e.g. for self-signed certificates.) 

**Smartphone (please complete the following information):**
 - Device brand/model: Pixel 6a
 - Android version: 13

**Comment by phpbg** [comment]
This should be fixed in #12 

Please either wait for v1.4 release on google playstore (within few hours), or [download the latest version here](https://github.com/phpbg/easysync/releases/tag/v1.4).

* If it works on chrome **on your phone**, then it should work with the new release.
* Otherwise, you may have to add your certificate within android user certificates in `Settings` > `Encryption and Credentials`.

Please let me know if this works for you.

**Comment by phpbg** [comment]
See #39
